cmd/probe/eventhandler.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package probe
+
+import (
+	"github.com/golang/glog"
+	"github.com/openebs/node-disk-manager/cmd/controller"
+)
+
+// UdevEventMessage channel used to send event message
+var UdevEventMessage = make(chan EventMessage)
```

Comment:
Can this be renamed? Also, is this going to be only for udev event message? Or can this also be used for other probe event messages?

Reply:
`udevprobe.listen()` uses this channel to get event and later udev monitor will use to send event details to udev probe. so this channel is udev specific. I will put it in udevprobe.go file.